packages/sanity/src/core/preview/components/Preview.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import {RenderPreviewCallbackProps} from '../../form/types'
+import {usePreviewComponent} from '../../form/form-components-hooks'
+import {PreviewLoader} from '../index'
+
+/**
+ * @beta
+ */
+export function Preview(props: RenderPreviewCallbackProps) {
+  const PreviewComponent = usePreviewComponent()
```

Comment:
There was a couple of places that we didn't call the `renderPreview()`-callback but instead used a preview component directly, but it was using a preview "view" component and did not go through the PreviewLoader, so it would not work (as these "view"-components could not take a raw form `value` and turn it into prepared preview fields). So this one serves as a component you can pass a `value` to and have the preview fields resolved and passed to the configured preview component for that type. Hope that makes sense :)